src/s1reader/s1_orbit.py

```diff
@@ -131,8 +182,6 @@ def get_file_name_tokens(zip_path: str) -> [str, list[datetime.datetime]]:
     Returns
     -------
     mission_id: ('S1A', 'S1B')
-    orbit_path : str
```

Comment:
(sorry for direct pushing to your branch, this docstring just confused me when i was looking through the code)